Gordon360/Gordon360.csproj

```diff
@@ -452,18 +452,6 @@
     <Compile Include="Models\MYSCHEDULE_BY_ID_Result.cs">
       <DependentUpon>CCT_DB_Models.tt</DependentUpon>
     </Compile>
-    <Compile Include="Models\NEWS_CATEGORIES_Result.cs">
-      <DependentUpon>CCT_DB_Models.tt</DependentUpon>
-    </Compile>
-    <Compile Include="Models\NEWS_NEW_Result.cs">
-      <DependentUpon>CCT_DB_Models.tt</DependentUpon>
-    </Compile>
-    <Compile Include="Models\NEWS_NOT_EXPIRED_Result.cs">
-      <DependentUpon>CCT_DB_Models.tt</DependentUpon>
-    </Compile>
-    <Compile Include="Models\NEWS_PERSONAL_UNAPPROVED_Result.cs">
```

Comment:
Interesting... I think those got removed when I did a merge from apartapp-edit-application, which doesn't make sense since the base of this branch is off of apartapp-edit-application, which makes it more bizarre...

Reply:
After a long process of manually fixing problems caused by merge conflicts, I think this part should now be fixed (I tested it on all the branches to see that they could build and run without error before committing each merge, unlike the last time I did a merge